crates/forge/src/fuzzer.rs

```diff
@@ -0,0 +1,171 @@
+use cairo_felt::Felt252;
+use num_bigint::{BigUint, RandBigInt};
+use num_traits::One;
+use rand::rngs::StdRng;
+use rand::{Rng, SeedableRng};
+use std::default::Default;
+
+#[derive(Default)]
+#[allow(clippy::module_name_repetitions)]
+pub struct FuzzerRunParams {
+    /// Inclusive value
+    low: BigUint,
+    /// Exclusive value
+    high: BigUint,
+    /// Number of arguments
+    args_number: usize,
+    /// Total number of runs
+    runs_number: u32,
+    /// Number of already executed runs
+    runs_executed: u32,
+    /// Runs in which arguments have min value e.g. `run_with_min_value[0] = 5`
+    /// means that the first argument has the lowest possible value in 5th run
+    runs_with_min_value: Vec<u32>,
+    /// Runs in which arguments have max value e.g. `run_with_min_value[0] = 5`
+    /// means that the first argument has the highest possible value in 5th run
+    runs_with_max_value: Vec<u32>,
```

Comment:
The point is to always have at least one run for each arg where it is set to max and min value, it was requested by @MaksymilianDemitraszek

Reply:
I get the idea, but perhaps it can be handled in less complicated way than creating these vectors. Something like just a counter and if in `next_felt252`.